docs/installation.md

```diff
@@ -41,9 +41,12 @@ services:
     ports:
       - 8080:80
     environment:
-      - GOTIFY_DEFAULTUSER_PASS=custom
+      GOTIFY_DEFAULTUSER_PASS: 'my-com"plex.\password'
     volumes:
       - "./gotify_data:/app/data"
+    # to run gotify as a dedicated user:
+    # sudo chown -R 1234:1234 ./gotify_data
+    user: "1234:1234"
```

Comment:
Can we comment this out? I'd rather have something that works out of the box, so we don't get the noobie questions how to change unix permissions etc. Users that read what they copy can then uncomment this line.